cpp/src/copying/split.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 BlazingDB, Inc.
```

Comment:
Update license.

Reply:
Should I change this  to Nvidia?

Reply:
No, there can be multiple copyrights if multiple parties worked on the file. We should put a shorthand NVIDIA copyright below the Blazing copyright.

Reply:
But, this was overwritten/completely new code. Should we still need to add Blazing ?

Reply:
Blazing doesn't need to be in the license anymore because the implementation is completely new.